modules/ROOT/nav.adoc

```diff
@@ -135,7 +126,7 @@ include::vector-search:partial$nav.adoc[]
   *** xref:fts:fts-advanced-settings-CBFT-ENV-OPTIONS.adoc[CBFT_ENV_OPTIONS]
  ** xref:fts:fts-monitor.adoc[Statistics and Monitoring]
  ** xref:fts:fts-troubleshooting.adoc[Troubleshooting and FAQs]
-* xref:eventing:eventing-overview.adoc[Eventing]
+* xref:eventing:eventing-overview.adoc[Run a Function on Data Change]
```

Comment:
@simon-dew  In Capella, this nav entry is "Add Eventing to your application" (Which I happen to like better). Any reason it's different here?

Reply:
This was the title from Dina's IA plan, as discussed with Jon Strabala, which I don't think was ever implemented.

Reply:
would like to revisit this at some point..but not today...